strr-platform-web/app/components/connect/form/address/Index.vue

```diff
@@ -40,100 +37,146 @@ const addressComplete = () => {
   }
 }
 
+const addId = useId()
 </script>
-
 <template>
   <div data-testid="form-section-mailing" class="space-y-3">
-    <div class="flex flex-row">
-      <UFormGroup v-slot="{ error }" :name="schemaPrefix + 'country'" class="grow">
+    <!-- country menu -->
+    <UFormGroup :name="schemaPrefix + 'country'" class="grow">
+      <template #default="{ error }">
         <USelectMenu
           v-model="country"
           value-attribute="alpha_2"
           class="w-full"
           size="lg"
           :color="country ? 'primary' : 'gray'"
           :placeholder="$t('label.country')"
+          :aria-label="$t('label.country')"
           :options="countries"
           :ui-menu="{ label: country ? 'text-gray-900' : !!error? 'text-red-600': 'text-gray-700'}"
           option-attribute="name"
           data-testid="address-country"
+          :aria-required="true"
+          :aria-invalid="error !== undefined"
+          :aria-describedby="schemaPrefix + 'country-' + addId"
         />
-      </UFormGroup>
-    </div>
-    <div class="flex flex-row">
-      <UFormGroup :name="schemaPrefix + 'street'" :hint="t('text.streetHint')" class="grow">
+      </template>
+
+      <template #error="{ error }">
+        <span :id="schemaPrefix + 'country-' + addId">
```

Comment:
just curious why the id is needed here and below? Is this for an AA thing?

Reply:
Similar to the label element where you tie a label to an input by an id, you can specify additional details with the [aria-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby) attribute. So adding this id and matching it to the aria-describedby on the input tells the screen reader user there is additional info for whatever input they're on.